hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/safemode/SCMSafeModeManager.java

```diff
@@ -319,6 +323,14 @@ public void setPreCheckComplete(boolean newState) {
     this.preCheckComplete.set(newState);
   }
 
+  public boolean isSafeModeExitForceful() {
+    return forceExitSafeMode.get();
+  }
```

Comment:
the name of the variable is ``forceExitSafeMode`` the setter is also ``setForceExitSafeMode`` can we keep this method name in line: ``isForceExitSafeMode()``

Reply:
Done.